pkg/collector/corechecks/oracle/statements_test.go

```diff
@@ -126,3 +126,49 @@ func TestUInt64Binding(t *testing.T) {
 		assert.Equalf(t, retValue, 1, "IN uint64")
 	}
 }
+
+func TestLongMultibyteQuery(t *testing.T) {
```

Comment:
It would be better to place the test in `activity_integration_test.go` as we are testing query samples, not the query metrics.